packages/workbox-build/src/entry-points/generate-sw.js

```diff
@@ -55,7 +51,7 @@ async function generateSW(input) {
     process.env.NODE_ENV.startsWith('dev')) ? 'dev' : 'prod';
 
   if (options.importWorkboxFromCDN) {
-    const cdnUrl = useBuildType(CDN_URL, buildType);
+    const cdnUrl = cdnUtils.getModuleUrl('workbox-sw', buildType);
```

Comment:
I don't know if there is ever a need to use the dev build of workbox-sw.

`new WorkboxSW({debug: true | false})` is the intended path to switch between dev and prod. workbox-sw is just a thin wrapper.